server/game/cards/13.4-BtRK/TunnelsOfTheRedKeep.js

```diff
@@ -0,0 +1,23 @@
+const DrawCard = require('../../drawcard.js');
+
+class TunnelsOfTheRedKeep extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Kneel and return to shadows',
+            cost: ability.costs.kneelSelf(),
+            limit: ability.limit.perPhase(1),

```

Comment:
Should be `max: ability.limit.perPhase(1)` instead of limit here (per the card).